doc/release_notes/release_notes_1.5.rst

```diff
@@ -0,0 +1,328 @@
+.. _release_notes_1.5:
+
+ACRN v1.5 (Jan 2020)
+####################
+
+We are pleased to announce the release of ACRN version 1.5.
+
+ACRN is a flexible, lightweight reference hypervisor that is built with
+real-time and safety-criticality in mind. It is optimized to streamline embedded
+development through an open source platform. Check out the :ref:`introduction` for more information.
+All project ACRN source code is maintained in the https://github.com/projectacrn/acrn-hypervisor
+repository and includes folders for the ACRN hypervisor, the ACRN device
+model, tools, and documentation. You can either download this source code as
+a zip or tar.gz file (see the `ACRN v1.5 GitHub release page
+<https://github.com/projectacrn/acrn-hypervisor/releases/tag/v1.5>`_)
+or use Git clone and checkout commands::
+
+   git clone https://github.com/projectacrn/acrn-hypervisor
+   cd acrn-hypervisor
+   git checkout v1.5
+
+The project's online technical documentation is also tagged to correspond
+with a specific release: generated v1.5 documents can be found at https://projectacrn.github.io/1.5/.
+Documentation for the latest (master) branch is found at https://projectacrn.github.io/latest/.
+ACRN v1.5 requires Clear Linux* OS version 31XXX. Follow the
+instructions in the :ref:`rt_industry_setup`.
+
+Version 1.5 major features
+**************************
+
+What's New in v1.5
+==================
+* ACRN now conforms to the Microsoft* Hypervisor Top-Level Functional Specification (TLFS).
+* ACRN scheduler framework re-architected capabilities have been added.
+* WaaG (Windows as a guest) stability and performance has been improved.
+* Realtime performance of the RTVM (preempt-RT kernel-based) has been improved.
+
+Document updates
+================
+Many new `reference documents <https://projectacrn.github.io>`_ are available, including:
+
+* :ref:`ACRN high-level design <hld>` documents.
+* :ref:`enable-s5`
+* :ref:`enable_laag_secure_boot`
+* :ref:`How-to-enable-secure-boot-for-windows`
+* :ref:`asa`
+
+Security Vulnerabilities
```

Comment:
@NanlinXie The "Security Vulnerabilities" section was included in v1.4. Does it need to be included again?